cypress.json

```diff
@@ -2,7 +2,7 @@
   "chromeWebSecurity": false,
   "integrationFolder": "./e2e/spec",
   "screenshotsFolder": "./e2e/screenshots",
-  "baseUrl": "http://localhost:8080/",
+  "baseUrl": "http://localhost:5000/",
```

Comment:
Are you running demo at `5000` now? I dont see that change.

Reply:
"start": "yarn serve -S demo"

[serve](https://www.npmjs.com/package/serve) uses 5000 by default